felix/wireguard/wireguard.go

```diff
@@ -150,7 +150,7 @@ type Wireguard struct {
 	cidrToNodeName map[ip.CIDR]string
 
 	// Wireguard routing table and rule managers
-	routetable *routetable.RouteTable
+	routetable routeTable
```

Comment:
Not sure about the change here.

Reply:
See the other comment, changed `routetable.RouteTable` struct with an `wireguard.routeTable` interface.